packages/cursorless-vscode-core/commands.ts

```diff
@@ -0,0 +1,11 @@
+import * as vscode from "vscode";
```

Comment:
added this file to keep general VSCode commands that don't really belong anywhere else